TombEngine/Objects/game_object_ids.h

```diff
@@ -374,6 +374,8 @@ enum GAME_OBJECT_ID : short
 	ID_THOR_HAMMER_HANDLE,
 	ID_THOR_HAMMER_HEAD,
 	ID_MOVING_LASER,
+	ID_TURNING_WALLBLADE,
```

Comment:
Please use `WALL_BLADE` and `CEILING_BLADE` everywhere. Also please don't forget to define those objects on TE side in catalogs (along with with TR3 remaps).

Reply:
I suggest that for new object slots, we separate words using underline symbols in any case (old slots which already known for a long time may stay as is though)